src/components/scenes/SendConfirmationScene.js

```diff
@@ -223,6 +218,7 @@ export class SendConfirmation extends React.Component<Props, State> {
     const displayAddress = fioAddress ? '' : address
 
     const feeCalculated = !!networkFee || !!parentNetworkFee
+    const showSpinner = !(feeCalculated || errorMsg || nativeAmount === '0' || !this.state.showSpinner || !this.props.pending)
```

Comment:
If we invert this, we get `const showSpinner = !feeCalculated && !errorMsg && nativeAmount !== '0' && this.state.showSpinner && this.props.pending`, but before it was `... this.state.showSpinner || this.props.pending`, so we have an `&&` turned to a `||`, which is a change in logic.